src/routes/current-studies/Content.svelte

```diff
@@ -31,6 +20,22 @@ function leaveStudy(studyID) {
     dispatch("leave-study", studyID);
 }
 
+function parseDateIfNeeded(date) {
```

Comment:
JS date parsing is terrible :)

Reply:
indeed :) hope this work around suffices for now. If we maintain a core add-on I want to integrate ts here.